components/access-management/kfam/api_default.go

```diff
@@ -248,9 +248,7 @@ func (c *KfamV1Alpha1Client) ReadBinding(w http.ResponseWriter, r *http.Request)
 		return
 	}
 	IncRequestCounter("", "", action, r.URL.Path)
-	if writeResponse(w, result) {
-		w.WriteHeader(http.StatusOK)
-	} else {
+	if !writeResponse(w, result) {
```

Comment:
200 is sent off by default if writeHeader is not called, so no reason to call writeheader(200) again, this will still be superfluous regardless.